providers/providers.go

```diff
@@ -61,9 +62,24 @@ func init() {
 
 	LastProviderInstall = time.Now().Unix()
 
-	// Initialize the global coordinator instance
-	coordinator := newCoordinator()
-	Coordinator = coordinator
+	viper.SetEnvPrefix("mondoo")
+	viper.AutomaticEnv()
+
+	autoUpdateEnabled := true
+	config.InitViperConfig()
```

Comment:
viper should not be even imported here. This should not be used outside of the cmd package